packages/shared/src/components/modals/SquadMemberModal.tsx

```diff
@@ -41,28 +47,54 @@ export function SquadMemberModal({
     },
   );
 
+  const onReportClick = (e, userId) => {
+    setMemberId(userId);
+    onMenuClick(e);
+  };
+
   return (
-    <UserListModal
-      {...props}
-      size={ModalSize.Medium}
-      title="Squad members"
-      scrollingProps={{
-        isFetchingNextPage: queryResult.isFetchingNextPage,
-        canFetchMore: checkFetchMore(queryResult),
-        fetchNextPage: queryResult.fetchNextPage,
-      }}
-      users={queryResult.data?.pages
-        .map((p) => p.sourceMembers.edges.map(({ node }) => node.user))
-        .flat()}
-      // additionalContent={(user) => (
-      //   <Button
-      //     buttonSize="small"
-      //     className="m-auto mr-0 btn-tertiary"
-      //     iconOnly
-      //     icon={<MenuIcon />}
-      //   />
-      // )}
-    />
+    <>
+      <UserListModal
+        {...props}
+        size={ModalSize.Medium}
+        title="Squad members"
+        scrollingProps={{
+          isFetchingNextPage: queryResult.isFetchingNextPage,
+          canFetchMore: checkFetchMore(queryResult),
+          fetchNextPage: queryResult.fetchNextPage,
+        }}
+        users={queryResult.data?.pages
+          .map((p) => p.sourceMembers.edges.map(({ node }) => node.user))
+          .flat()}
+        additionalContent={(user) => {
+          const role = queryResult.data?.pages
+            .map((p) =>
```

Comment:
```
p -> page
```

We had a rule before to never use letters as a variable name before. I think it makes sense